packages/block-library/src/quote/deprecated.js

```diff
@@ -34,6 +35,76 @@ export const migrateToQuoteV2 = ( attributes ) => {
 	];
 };
 
+const TEXT_ALIGN_OPTIONS = [ 'left', 'right', 'center' ];
+
+// Migrate existing text alignment settings to the renamed attribute.
+const migrateTextAlign = ( attributes ) => {
```

Comment:
I haven't checked the code but if we add a migration, we need to add it in all existing deprecations

Reply:
I was not able to figure out how to do this. The deprecations each use different logic for the existing `migrate` (migrateToQuoteV2), I am not sure how to combine the two migrations in `v0` and `v1`.

Reply:
In general you combine all the migration actions into a new `migrate` function for each deprecation. This might need more eyes though because it's being called in `edit` as well. --cc @oandregal @ellatrix